graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -77,9 +78,16 @@ def _get_fields_for_class(schema_graph, graphql_types, field_type_overrides, hid
         property_name: graphql_type
         for property_name, graphql_type in six.iteritems(all_properties)
         if not (isinstance(graphql_type, GraphQLList) and
-                isinstance(graphql_type.of_type, GraphQLObjectType))
+                not isinstance(graphql_type.of_type, GraphQLScalarType))
     }
 
+    class_collection_properties = set(all_properties.keys()).difference(result.keys())
+    if len(class_collection_properties) > 0:
+        warnings.warn('The fields {} of class {} were ignored since they are GraphQLLists of '
```

Comment:
Sorry, I don't see how the warning is inaccurate. It seems to me that that is exactly what the code above does.

Reply:
Ah yes I think you are right, my bad. I was confused because we first capture all properties, then select a subset of them with an `if not`, and then we do set difference. This is a bit convoluted and we may be able to simplify it. How about refactoring so that we explicitly select the properties we want to exclude (i.e. directly compute `class_collection_properties`) and then compute the result by excluding the unsupported properties from the all-properties dict?

Reply:
Will add this change in next pr update.